api/v1alpha1/bundle_types.go

```diff
@@ -64,6 +65,8 @@ type BundleSource struct {
 	Git *GitSource `json:"git,omitempty"`
 	// Local is a reference to a local object in the cluster.
 	Local *LocalSource `json:"local,omitempty"`
+	//  HTTP is the remote location that backs the content of this Bundle.
+	HTTP *HTTPSource `json:"http,omitempty"`
```

Comment:
Maybe this should be `HTTPS` if most charts being referenced are from a domain secured by a certificate? And we want to encourage people to use https in general?

Reply:
OK. I'll change HTTP to HTTPS everywhere.

Reply:
The scheme will ultimately be determined by the URL though right? Will we accept `http://` URLs? I could imagine a use case where the bundle is hosted by another pod in the namespace, or perhaps in some wild configuration in a container that's a sidecar of the helm provisioner (in which case `http://localhost:8080/my-chart.tgz` might be the URL)

Reply:
Yes, the scheme is determined by the URL and it can accept http URLs.